components/math/longestStreak/Winner.tsx

```diff
@@ -1,36 +1,118 @@
-import React, { useEffect, useState } from "react";
-import { Button } from "../../ui/Button";
+import { useMutation } from "@apollo/client";
+import React, { useState } from "react";
+import { RESET_GAME_LEVEL } from "../../../graphql/longestStreak/resetGameLevel";
 
 export interface WinnerProps {
   text: string;
   onClick: () => void;
+  onRestartClick: () => void;
+  onSameLevelClick: () => void;
   winner: string;
-  image: any;
+  image: string;
+  user: any;
 }
 
 export const Winner: React.FC<WinnerProps> = ({
   text,
   onClick,
+  onRestartClick,
+  onSameLevelClick,
   winner,
   image,
+  user,
   ...props
 }) => {
+  const [resetGameLevel] = useMutation(RESET_GAME_LEVEL);
+
+  function handleResetGameLevel() {
+    resetGameLevel({
+      variables: {
+        userId: user.uid,
+      },
+      refetchQueries: [
+        {
+          query: RESET_GAME_LEVEL,
```

Comment:
Why are you refetching this mutation here? I think only queries should be refetched

Reply:
Unless this is supposed to be fetch game level?

Reply:
this i removed and put in the game.tsx